load_masks.py

```diff
@@ -0,0 +1,200 @@
+            if max_value < mean:
+                mean = max_value
+                t1,t2 = task_1, task_2
+    print('MEAN ', mean)
+    print(t1, ' ',t2)
+
+    return stats_jaccard_task_dict
+
+def jaccard_similarity_per_lang_pair(masks_dict):
+    '''
+        same task take language pairs 
+
+        Parameters:
+            masks_dict: dict of the form {language: { task: {seed: 2d_mask}}}
+        Returns:
+            stats_jaccard_lang_dict: dict of the form {task: { (language1, language2): 'mean_jaccard+-std_jaccard'}}
+    '''
+    stats_jaccard_lang_dict = {}# mean+-std across seeds
+    print('LANGUAGE PAIRS')
+    max_value = -1
+    for task in TASKS:
+
+        stats_jaccard_lang_dict[task] = {}
+
+        for lang_1, lang_2 in itertools.combinations(LANGUAGES, r=2):
+            tmp_list_jaccard_seeds = []
+            for seed in range(NUM_SEEDS):
+                #! pass same task, same seed but different languages
+                mask_1 = masks_dict[lang_1][task][seed].flatten().tolist()
+                mask_2 = masks_dict[lang_2][task][seed].flatten().tolist()
+
+                tmp_jaccard_sim = compute_jaccard_similarity(mask_1, mask_2)
+                tmp_list_jaccard_seeds.append(tmp_jaccard_sim)
+
+            mean = round(statistics.mean(tmp_list_jaccard_seeds),3)
+            std = round(statistics.stdev(tmp_list_jaccard_seeds),3)
+            stats_jaccard_lang_dict[task][(lang_1,lang_2)] = f'{mean}\u00B1{std}'# mean +- std
+            if max_value < mean:
+                mean = max_value
+                t1,t2 = lang_1, lang_2
+    print('MEAN ', mean)
+    print(t1, ' ',t2)
+
+    return stats_jaccard_lang_dict
+
+def jaccard_similarity_per_seed_pair(masks_dict):
+    '''
+        same task same language, different seed 
+        Parameters:
+            masks_dict: dict of the form {language: { task: {seed: 2d_mask}}}
+        Returns:
+            stats_jaccard_seed_dict: dict of the form {task: {language: {(seed_i, seed_j) : 'jaccard'}}}
+    '''
+    jaccard_seed_dict = {} 
+    stats_jaccard_seed_dict = {}
+    for task in TASKS:
+        jaccard_seed_dict[task] = {}
+        stats_jaccard_seed_dict[task] = {}
+
+        for lang in LANGUAGES:
+            jaccard_seed_dict[task][lang] = {}
+            tmp_list_jaccard_seeds = []
+            for seed_i, seed_j in itertools.combinations(SEEDS, r=2):
+                #! pass same task, same language but different seeds
+                mask_1 = masks_dict[lang][task][seed_i].flatten().tolist()
+                mask_2 = masks_dict[lang][task][seed_j].flatten().tolist()
+                tmp_jaccard_sim = compute_jaccard_similarity(mask_1, mask_2)
+
+                jaccard_seed_dict[task][lang][(seed_i,seed_j)] = tmp_jaccard_sim
+                tmp_list_jaccard_seeds.append(tmp_jaccard_sim)
+
+            mean = round(statistics.mean(tmp_list_jaccard_seeds),3)
+            std = round(statistics.stdev(tmp_list_jaccard_seeds),3)
```

Comment:
Is there a reason we imported a new module here? Is statistics a default python module?
(my first intention on this would be np.mean and np.std)